mitodl/ocw-hugo-themes
# add course download page link button

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/938

#### What's this PR do?
This PR finalizes the work done on course downloads and makes the feature public by adding the buttons to the download page on the course home page and the course info drawer. The link points to `/download`, taking `baseUrl` into account.

#### How should this be manually tested?
 - Configure your `.env` file to spin up a test course (preferably one with a variety of resources) and do so with `yarn start:course`
 - Visit http://localhost:3000 and ensure that the download button on the right side looks correct and links to the download page
 - Visit any other page in the nav (syllabus, lecture notes, etc.) and verify that the link in the course info drawer also looks correct and links to the download page
 - Shut down the dev server and make a temporary edit to `package_scripts/start_course.sh`, adding `--baseUrl http://localhost:3000/test/ \` to the `hugo server` arguments list
 - Visit http://localhost:3000/test/ and follow the same procedure, ensuring that the links all go where they should

#### Screenshots (if appropriate)
![chrome_2022-11-09_01-09-07](https://user-images.githubusercontent.com/12089658/200752828-705645d1-c5a6-433f-8e3b-63980712bb69.png)
![chrome_2022-11-09_01-09-30](https://user-images.githubusercontent.com/12089658/200752859-cf79bc16-8dae-42b7-8b83-addc5a298391.png)
![chrome_2022-11-09_01-09-59](https://user-images.githubusercontent.com/12089658/200752943-627ae57f-6711-4eef-b023-1c49ec95d0b3.png)
![chrome_2022-11-09_01-10-17](https://user-images.githubusercontent.com/12089658/200752984-dcddf756-49cf-455a-88f7-fa3ff9d475d2.png)
